spec/acceptance/server/grant_spec.rb

```diff
@@ -0,0 +1,190 @@
+require 'spec_helper_acceptance'
+
+describe 'postgresql::server::grant:', :unless => UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+
+  db = 'grant_priv_test'
+  owner = 'psql_grant_priv_owner'
+  user = 'psql_grant_priv_tester'
+  password = 'psql_grant_role_pw'
+
+  pp_setup = <<-EOS.unindent
```

Comment:
This was done for consistency with other spec tests (as you spotted).

Happy to change to something more sane if you want though - just let me know.